frontend/src/js/standard-query-editor/QueryNode.tsx

```diff
@@ -127,49 +133,51 @@ const QueryNode = ({
 
     ...(nodeIsConceptQueryNode(node)
       ? {
-          ids: node.ids,
-          type: node.type,
-          description: node.description,
-          tree: node.tree,
-          tables: node.tables,
-          selects: node.selects,
-
-          additionalInfos: node.additionalInfos,
-          matchingEntries: node.matchingEntries,
-          matchingEntities: node.matchingEntities,
-          dateRange: node.dateRange,
-        }
+        ids: node.ids,
+        type: node.type,
+        description: node.description,
+        tree: node.tree,
+        tables: node.tables,
+        selects: node.selects,
+
+        additionalInfos: node.additionalInfos,
+        matchingEntries: node.matchingEntries,
+        matchingEntities: node.matchingEntities,
+        dateRange: node.dateRange,
+      }
```

Comment:
I added a [CI step for lint & checking format](https://github.com/bakdata/conquery/pull/2856). Could you please merge develop into this branch and fix the formatting issues until the pipeline is green?